migrations/20200917163300-create-events.js

```diff
@@ -0,0 +1,22 @@
+exports.up = (db, callback) => db.createTable('event', {
+  id: { type: 'int', primaryKey: true, autoIncrement: true },
+  type: 'string',
+  label: 'string',
```

Comment:
should `type` and `label` disallow `null`s as well?

Reply:
@davemcorwin  - my thought was that these would be req'd fields ... have you thought of scenarios where we should keep these blank?

Reply:
I said **dis**allow... :)

Reply:
hah 😄 ... it is enforced in the model but will add here too 👍